_tests_/pet.test.js

```diff
@@ -0,0 +1,140 @@
+const Pet = require('../src/pet');
+
+
+describe('constructor', () => {
+    it('returns an object', () => {
+        expect(new Pet('Fido')).toBeInstanceOf(Object);
+    });
+    it('checks initial age of pet', () => {
+        const pet = new Pet('Fido');
+        expect(pet.age).toBe(0);
+    });
+});
+    describe('Declare Pets', () => {
+        let pet;
+        let parent;
+        let child;
+        beforeEach(() => {
+            pet = new Pet('Fido');
+            parent = new Pet('Pal');
+            child = new Pet('Pololo');
+        });
+    describe('test the growUp function', () => {
+        it('increments age of the pet by 1', () => {
+            pet.growUp();
+            
+            expect(pet.age).toEqual(1);
+        });
+        it('checks how growUp modifies fitness and hunger', () => {
+            pet.growUp();
+            
+            expect(pet.fitness).toEqual(7);
+            expect(pet.hunger).toEqual(5);
+           
+            pet.growUp();
+            
+            expect(pet.fitness).toEqual(4);
+            expect(pet.hunger).toEqual(10);
+        });
+    });
+    describe('The use of the walk function', () => {
+        it('increases fitness and should not be over 10', () => {
+            pet.fitness = 10;
+            pet.walk();
+            expect(pet.fitness).toEqual(10);
```

Comment:
Its common convention to leave a clear line before the start of the `expect` assertions, should add one here as well as on line 48